invenio_files_rest/alembic/8ae99b034410_create_files_objecttags_table.py

```diff
@@ -43,7 +44,9 @@ def upgrade():
         "files_objecttags",
         sa.Column("version_id", sqlalchemy_utils.types.uuid.UUIDType(), nullable=False),
         sa.Column(
-            "key", sa.Text().with_variant(mysql.VARCHAR(255), "mysql"), nullable=False
+            "key",
+            sa.String(255).with_variant(mysql.VARCHAR(255), "mysql"),
```

Comment:
```suggestion
            sa.Text().with_variant(mysql.VARCHAR(255), "mysql"),
```

I think this has to stay as `sa.Text()` since there's an expectation that in PostgreSQL you can have a string as long as one needs.

Reply:
the problem is that the type in the alembic recipe and the type in the model are not the same.

model has db.String(255) [here](https://github.com/inveniosoftware/invenio-files-rest/blob/8e09e6152c6b195ce439f6a665624a2ddd274dd5/invenio_files_rest/models.py#L1461)
alembic recipe has db.TEXT [here](https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/alembic/8ae99b034410_create_files_objecttags_table.py#L46)

older alembic or sqlalchemy didn't see a problem with that, but now we have one

Reply:
Yikes, then yes that was a mistake, so this fix is valid :)